phaseshifts/lib/phsh.py

```
@@ -870,27 +842,23 @@ def phsh_rel(
 
     inpdat.write("input data\n")  # 13 format (1h1, /  / ,t61,'input data',)
     name = mtz.readline().split()  # 10
-    (es, de, ue, lsm, vc) = [
-        t(s) for t, s in zip((float, float, float, int, float), mtz.readline().split())
-    ]
+    (es, de, ue, lsm, vc) = [t(s) for t, s in zip((float, float, float, int, float), mtz.readline().split())]
```

Ruff (0.14.10)

Unpacked variable `vc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)